src/mono/mono/utils/mono-time.c

```diff
@@ -229,3 +233,48 @@ mono_100ns_datetime_from_timeval (struct timeval tv)
 
 #endif
 
+#ifdef HOST_DARWIN
+
+static clock_serv_t sampling_clock_service;
```

Comment:
This used to be initialized in `clock_init()` under `mono-posix.c`, but it's now always `0`.